elements-code-tutorial/easy-run-code.md

```diff
@@ -230,9 +230,10 @@ e1-cli generatetoaddress 1 $ADDRGEN1
 sleep 10
 e2-cli listissuances
 
-IADDR=$(e1-cli gettransaction $ITXID | jq -r '.details[0].address')
-
-e2-cli importaddress $IADDR
+ISSUE_RAW_TX=$(e2-cli getrawtransaction $ITXID 1)
+ISSUE_VOUTS=$(echo $ISSUE_RAW_TX | jq -r '.vout')
+VOUT_ADDRESS_ISSUE=$(echo $ISSUE_VOUTS | jq -r '.[0].scriptPubKey.addresses[0]')
+e2-cli importaddress $VOUT_ADDRESS_ISSUE
```

Comment:
I don't know if the other effects of importing the address might be annoying here for the user. This might mean that fundrawtransaction (perhaps with include_watchonly set to true) to include UTXOs sent to that address.

Reply:
What other way is there for another node to know about an issuance a node made? Do you know?

Reply:
Tbh I don't think there is a way to do that explicitly. So yeah this seems like the best guess. If you think this is a feature worth having, feel free to open an issue on the elements github. I'm not entirely sure how hard it would be.